server/src/internalClusterTest/java/org/opensearch/discovery/StableMasterDisruptionIT.java

```diff
@@ -228,9 +228,9 @@ public void testStaleMasterNotHijackingMajority() throws Exception {
                         event.state(),
                         event.previousState()
                     );
-                    String previousMasterNodeName = previousMaster != null ? previousMaster.getName() : null;
+                    String previousClusterManagerNodeName = previousMaster != null ? previousMaster.getName() : null;
```

Comment:
`previousMaster` -> `previousClusterManager`?

Reply:
Changed the variable name. 👍